README.md

```diff
@@ -1,2 +1,51 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # To_DO_List
 This the minimal to do list done using JavaScript, html and CSS
+
+> In this project, we built a basic to do list website: plain JavaScript with objects, that allows users to add/remove tasks from a list.
+
+![screenshot](./Capture.PNG)
+
+This to do list task 1 was done using Javascript, Html and Css by implementing webpack functionalities.
+
+## Built With
+
+- JavaScript (ES6)
+- HTML and CSS
+- Webpack
+
+**Instructions on setting up your project locally.**
+The project could be accessed by visiting the website link.
+
```

Comment:
- [ ] I would recommend adding all the steps to run the project locally. It should include How to clone, going to right folder, and running right command or opening right file to see the project.